src/models/task/types/taskPrisma.type.ts

```diff
@@ -0,0 +1,28 @@
+import { Status } from '@prisma/client';
+import {
+  Field, Float, ID, ObjectType,
+} from 'type-graphql';
+
+@ObjectType()
+export default abstract class ITaskPrisma {
+  @Field(() => ID)
+    id: string;
+
+  @Field(() => String)
+    subject: string;
+
+  @Field(() => String)
+    projectId?: string;
```

Comment:
```suggestion
    projectId: string;
```

Si on met le projectId en optionnel ici, ça ne correspond pas au `schema.prisma` et ça va casser